frontend/src/scenes/events/eventsTableLogic.ts

```diff
@@ -206,20 +229,20 @@ export const eventsTableLogic = kea({
             })
         },
         fetchEvents: [
-            async (_, breakpoint) => {
+            async ({}, breakpoint: (timeout: number) => void) => {
                 if (values.events.length > 0) {
                     await breakpoint(500)
                 }
                 if (values.isLoading === null) {
                     actions.setDelayedLoading()
                 }
             },
-            async ({ nextParams }, breakpoint) => {
-                clearTimeout(values.pollTimeout)
+            async ({ nextParams }: { nextParams: { before: string } }, breakpoint: () => void) => {
+                values.pollTimeout && clearTimeout(values.pollTimeout)
```

Comment:
@mariusandra how do you type this properly? I can't get this array of listeners to type without error

Reply:
I guess "array of listeners" is another non-implemented typegen issue. Made an issue here: https://github.com/keajs/kea-typegen/issues/14

I'll be glad to look at these once plugins start running on cloud! Could be quite soon... 😅

Reply:
Got it. It's not the biggest tragedy I was just wondering if I was misunderstanding how the types here work!